src/libraries/System.Private.CoreLib/src/System/Int64.cs

```diff
@@ -663,24 +663,11 @@ public static long Log2(long value)
         /// <inheritdoc cref="INumber{TSelf}.CopySign(TSelf, TSelf)" />
         public static long CopySign(long value, long sign)
```

Comment:
this method is currently not getting inlined and I can't see any perf gains.

## .NET 9.0.0 (42.42.42.42424), X64 RyuJIT AVX2
```assembly
; System.Tests.Perf_Int64.CopySign(Int64, Int64)
       mov       rcx,rdx
       mov       rdx,r8
       jmp       qword ptr [7FFF8C036118]; System.Int64.CopySign(Int64, Int64)
; Total bytes of code 12
```
```assembly
; System.Int64.CopySign(Int64, Int64)
       sub       rsp,28
       mov       rax,8000000000000000
       cmp       rcx,rax
       je        short M01_L01
M01_L00:
       xor       rdx,rcx
       sar       rdx,3E
       or        edx,1
       movsxd    rax,edx
       imul      rax,rcx
       add       rsp,28
       ret
M01_L01:
       test      rdx,rdx
       jl        short M01_L00
       call      qword ptr [7FFF8C57F6F0]
       int       3
; Total bytes of code 53
```

```suggestion
        [MethodImpl(MethodImplOptions.AggressiveInlining)]
        public static long CopySign(long value, long sign)
```